README.rst

```diff
@@ -39,40 +34,39 @@ Installation
 
 #. Add ``'captcha'`` to your ``INSTALLED_APPS`` setting.
 
-#. Add the keys reCAPTCHA have given you to your Django production settings (leave development settings blank to use the default test keys) as
-   ``RECAPTCHA_PUBLIC_KEY`` and ``RECAPTCHA_PRIVATE_KEY``. For example:
-
-   .. code-block:: python
-
-       RECAPTCHA_PUBLIC_KEY = 'MyRecaptchaKey123'
-       RECAPTCHA_PRIVATE_KEY = 'MyRecaptchaPrivateKey456'
+    .. code-block:: python
 
-   These can also be specificied per field by passing the ``public_key`` or
-   ``private_key`` parameters to ``ReCaptchaField`` - see field usage below.
+        INSTALLED_APPS = [
+            ...,
+            'captcha',
+            ...
+        ]
 
-#. By default the reCAPTCHA V2 -Checkbox widget will be used, to make use of the reCAPTCHA V2 - Invisible widget add the following setting:
+#. Add the keys reCAPTCHA have given you to your Django production settings (leave development settings blank to use the default test keys) as
```

Comment:
"has given"

Reply:
How about:
`Add the keys reCAPTCHA has given you to your Django production settings. Note that omitting these settings will default to a set of test keys which can be used for development.`
The sentence above seems too long with the part in brackets.